pkg/orchestration/wiring/entangler.go

```diff
@@ -135,14 +197,20 @@ func (en *Entangler) Entangle(state *orch_v1.State, context *EntangleContext) (*
 	for _, v := range sorted {
 		resource := g.Vertices[v].Data.(*orch_v1.StateResource)
 		dependants := getDependants(resource.Name, g.Vertices[v].IncomingEdges, state.Spec.Resources)
-		retriable, entErr := w.entangle(resource, &state.ObjectMeta, &stateContext, dependants)
+		external, retriable, entErr := w.entangle(resource, &state.ObjectMeta, &stateContext, dependants)
 		if entErr != nil {
-			return nil, retriable, errors.Wrapf(entErr, "failed to wire up resource %q of type %q", resource.Name, resource.Type)
+			return &EntangleResultFailure{
```

Comment:
As discussed earlier, it'd be great to not stop here but continue so that we detect more issues, not just the first one. Not in this PR, just saying. /cc @amckague